RELEASING.md

```diff
@@ -1,5 +1,18 @@
 # Releasing
 
-The process for how to release the crates in this repository are documented here:
+To release Soroban CLI, follow this process **in order**:
 
-https://github.com/stellar/actions/blob/main/README-rust-release.md
```

Comment:
The release process this repo should follow is still the linked rust release process, so I'm not sure this change is needed. The rust release process has instructions for how to create a release, which shouldn't be the git tag commands, but instead using the steps in the referenced markdown.